datavault-common/src/main/java/org/datavaultplatform/common/model/Permission.java

```diff
@@ -0,0 +1,39 @@
+package org.datavaultplatform.common.model;
+
+public enum Permission {
+
+    VAULT_PERMISSION_1(PermissionModel.PermissionType.VAULT, "Vault Permission 1"),
```

Comment:
Is `VAULT_PERMISSION_1` the literal identifier stored in the database? If so you might want to give them some other unique identifier that stays stable after renaming.

Reply:
That's a good point - if someone renamed one of these permissions in the code the database entry would get removed and a new default would be created causing any custom labels to disappear. I think I might pass an ID field in here as well